src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -139,15 +139,15 @@
         "Close"]]]]]])
 
 (defn- customizer-select-button
-  [{:keys [open selected-key]}]
+  [{:keys [open selected-option]}]
   [rn/pressable
    {:style    (style/select-container)
     :on-press #(reset! open true)}
    [rn/text
     {:style           (style/field-select)
      :number-of-lines 1}
-    (if selected-key
-      (humanize selected-key)
+    (if selected-option
+      (or (:value selected-option) (humanize (:key selected-option)))
```

Comment:
Thanks for fixing this 😄 🙏

Reply:
Sorry about the inconvenience. @ulisesmac reported this issue to me.